alerts/cluster-monitoring-operator/PrometheusOperatorRejectedResources.md

```diff
@@ -0,0 +1,89 @@
+# PrometheusOperatorRejectedResources
+
+## Meaning
+
+The `PrometheusOperatorRejectedResources` alert triggers when Prometheus Operator
+rejects invalid `Alertmanager`, `PodMonitor`, `ServiceMonitor`, or `PrometheusRules`
+objects.
+
+## Impact
+
+The custom resources that trigger the alert are ignored by Prometheus Operator.
+As a consequence, they will not be part of the final configuration of the
+Prometheus, Alertmanager, or Thanos Ruler components managed by Prometheus Operator.
+
+## Diagnosis
+
+Find the type of custom resource and its namespace in the `description` field
+of the `PrometheusOperatorRejectedResources` alert. You can find this information
+by using the OpenShift web console or the command line interface (CLI).
+
+### Diagnose using the Openshift web console
+
+1. Browse to **Observe** -> **Alerting**.
+2. The default **Filter** should be **Firing**. If not, click the **Filter** drop-down
```

Comment:
```suggestion
2. The default **Filter** for **Alert State** should be **Firing**. If not, click the **Filter** drop-down
```
see from
![alerts](https://github.com/juzhao/test/assets/22743651/6a4b85eb-eced-46b3-884d-984cb35d36ca)
there are also filters for Severity/Source in the drop-down list